cli/src/lib.rs

```diff
@@ -497,23 +490,7 @@ fn genesis_domain(public_key: PublicKey) -> Domain {
     domain
 }
 
-macro_rules! mutate_nested_option {
-    ($obj:expr, self, $func:expr) => {
-        $obj.as_mut().map($func)
-    };
-    ($obj:expr, $field:ident, $func:expr) => {
-        $obj.$field.as_mut().map($func)
-    };
-    ($obj:expr, [$field:ident, $($rest:tt)+], $func:expr) => {
-        $obj.$field.as_mut().map(|x| {
-            mutate_nested_option!(x, [$($rest)+], $func)
-        })
-    };
-    ($obj:tt, [$field:tt], $func:expr) => {
-        mutate_nested_option!($obj, $field, $func)
-    };
-}
-
+// FIXME: update docs
```

Comment:
reminder

Reply:
Also, to prevent reminders you left for yourself from compiling you should make them in a way they don't compile or pass our CI. For example, you could have added a failing doctest here. That would definitely remind you